tachyon/math/polynomials/univariate/univariate_evaluations.h

```diff
@@ -252,6 +253,29 @@ class Copyable<math::UnivariateEvaluations<F, MaxDegree>> {
   }
 };
 
+template <typename F, size_t MaxDegree>
+class RapidJsonValueConverter<math::UnivariateEvaluations<F, MaxDegree>> {
+ public:
+  template <typename Allocator>
+  static rapidjson::Value From(
+      const math::UnivariateEvaluations<F, MaxDegree>& value,
+      Allocator& allocator) {
+    rapidjson::Value object(rapidjson::kObjectType);
+    AddJsonElement(object, "evaluations", value.evaluations(), allocator);
+    return object;
+  }
+
+  static bool To(const rapidjson::Value& json_value, std::string_view key,
```

Comment:
```suggestion
  [[nodiscard]] static bool To(const rapidjson::Value& json_value, std::string_view key,
```

Reply:
Is it necessary to include the `nodiscard` keyword in utility functions? In my opinion, I'd prefer not to include it for user's convenience.